static/scripts/newblocks.js

```diff
@@ -739,6 +739,29 @@ Blockly.JavaScript["check_fitness"] = function (block) {
   return [code, Blockly.JavaScript.ORDER_FUNCTION_CALL];
 };
 
+
+Blockly.JavaScript["plotting_one_value"] = function (block) {
+  var variable_yValue = Blockly.JavaScript.valueToCode(
+    block,
+    "yValue",
+    Blockly.JavaScript.ORDER_ATOMIC
+  );
+  variable_datasetNumber = Blockly.JavaScript.valueToCode(
+    block,
+    "datasetNumber",
+    Blockly.JavaScript.ORDER_ATOMIC
+  );
+  variable_plotName =  block.getFieldValue("plotName");
+  variable_plotType  = block.getFieldValue("plotType"); 
```

Comment:
Why do you use `var` for `variable_yValue` and no descriptor for the other variables? :) Is there a reason?